app/src/main/java/com/io/moviO/movieDetails/MovieDetailsViewModel.kt

```diff
@@ -1,11 +1,19 @@
-package com.io.moviO
+package com.io.moviO.movieDetails
 
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import com.io.moviO.data.Movie
+import com.io.moviO.domain.GetMovieByIdUseCase
 
 class MovieDetailsViewModel : ViewModel() {
 
     private val _movie = MutableLiveData<Movie>()
     val movie: LiveData<Movie> = _movie
+
+    private val useCase = GetMovieByIdUseCase()
+
+    fun setMovie(id: String) {
```

Comment:
see - VM. setMovie calls UC. getMovieById 

You think the method sets things, while it actually gets things :D

Reply:
fixed